src/components/dashboard/components/dictionary/EditDictionary.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import DictionaryModal from './common/DictionaryModal';
+import {
+  fetchingOrganizations,
+  editDictionary,
+} from '../../../../redux/actions/dictionaries/dictionaryActionCreators';
+import languages from './common/Languages';
+
+export class EditDictionary extends React.Component {
+  preSelectDefaultLocale = () => {
+    const { dictionary } = this.props;
+    for (let i = 0; i < languages.length; i++) { // eslint-disable-line
```

Comment:
Hi @emmabaye 
Why are you disabling Eslint here?
```
// eslint-disable-line
```

Reply:
For flexibility, I need to use the unary operator.

Reply:
_`...For flexibility...`_ 😕 
I believe there is a reason why your team has configured Eslint to not allow unary operators. Considering the fact that you could have used an alternate approach that shouldn't cost you anything and would also make Eslint happy, don't you think this is against your team's convention?

Reply:
I've updated the code.